docs/rst/getting_started/usage_example.rst

```diff
@@ -0,0 +1,129 @@
+instances will establish communication after proper configuration and deployment of the two aforementioned routers.
+
+.. figure:: /rst/figures/shapesdemo_general.png
+
+.. note::
+
+    This example is valid both for LAN and WAN scenarios. For the WAN case, make sure that public IP addresses are used
+    instead, and that the provided ports are reachable by properly configuring port forwarding in your Internet router
+    devices.
+
+Launching ShapesDemo
+====================
+`ShapesDemo <https://www.eprosima.com/index.php/products-all/eprosima-shapes-demo>`_ is an application that publishes
+and subscribes to shapes of different colors and sizes moving on a board. This is nothing more than a graphical tool to
+test the correctness of a specific DDS protocol implementation, as well as to prove interoperability with other
+implementations.
+
+Let us launch a ShapesDemo instance in one of the DDS networks, and start publishing in topics ``Square``, ``Circle``
+and ``Triangle`` with default settings.
+
+.. figure:: /rst/figures/shapesdemo_publisher.png
+    :scale: 75 %
+
+Now, run another instance in the other network and subscribe to the same topics chosen in the publisher side (use
+default settings).
+
+.. figure:: /rst/figures/shapesdemo_subscriber.png
+    :scale: 75 %
+
+.. note::
+
+    If you are trying this example in the LAN scenario, make sure a different DDS domain id is used in one of the
+    ShapesDemo instances in order to avoid direct communication between them.
+
+Router configuration
+====================
+A configuration file is all that is required in order to run a |ddsrouter| instance. In a nutshell, each router will
+only forward messages if their associated topics are contained in its ``allowlist``. A ``blocklist`` can also be
+specified, on its own or in addition to an ``allowlist``, but we will not be covering this here.
+
+Let us first add only the ``Square`` topic:
+
+.. literalinclude:: ../../resources/getting_started/client-ddsrouter.yaml
+    :language: yaml
+    :lines: 3-6
+
+We must set ``keyed`` to ``true`` as shape topics use their color attribute as key. See
+:ref:`Topic Filtering <user_manual_configuration>` for more details on allowlisting.
+
+Apart from selecting on which topics we wish to send/receive data, we must configure as well the participants that will
+ultimately perform communication. Each router instance will contain a :ref:`simple <user_manual_participants_simple>`
+and a :ref:`wan <user_manual_participants_wan>` participants. In brief, the simple participants will be in charge of
+locally communicating with the corresponding ShapesDemo application, while the wan participants will be the ones
+bridging the connection between the two DDS networks.
+
+.. figure:: /rst/figures/shapesdemo_detailed.png
+
+The only configuration required for simple participants is the DDS ``domain`` identifier.
```

Comment:
NIT: Add subsections for LAN and WAN

Reply:
The example is valid both for LAN and WAN scenarios, with some small differences such as the one pointed out in the initial note.